generators/server/templates/src/main/docker/redis-cluster.yml.ejs

```diff
@@ -0,0 +1,86 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+version: '<%= DOCKER_COMPOSE_FORMAT_VERSION %>'
+services:
+    <%= baseName.toLowerCase() %>-redis-1:
+        image: <%= DOCKER_REDIS %>
+        command:
+            - "redis-server"
+            - "--port 7001"
+            - "--cluster-enabled yes"
+            - "--cluster-config-file nodes.conf"
+            - "--cluster-node-timeout 5000"
+            - "--appendonly yes"
+        ports:
+            - "7001:7001"
+    <%= baseName.toLowerCase() %>-redis-2:
+        image: <%= DOCKER_REDIS %>
+        command:
+            - "redis-server"
+            - "--port 7002"
+            - "--cluster-enabled yes"
+            - "--cluster-config-file nodes.conf"
+            - "--cluster-node-timeout 5000"
+            - "--appendonly yes"
+        ports:
+            - "7002:7002"
+    <%= baseName.toLowerCase() %>-redis-3:
+        image: <%= DOCKER_REDIS %>
+        command:
+            - "redis-server"
+            - "--port 7003"
+            - "--cluster-enabled yes"
+            - "--cluster-config-file nodes.conf"
+            - "--cluster-node-timeout 5000"
+            - "--appendonly yes"
+        ports:
+            - "7003:7003"
+    <%= baseName.toLowerCase() %>-redis-4:
```

Comment:
maybe 3 nodes is enough?

Reply:
good question, honestly I don't know. I'm not a Redis expert at all, I simply saw that in several docker-compose files, that's why I did the same but without using network